### PR TITLE
fix: pre-commit conflicting deps

### PR DESCRIPTION
**Pull request by NiceAesth**
# Describe your changes

`blacken-docs` was requesting a version of black that is out of date.

## Related Issues / Projects

This PR fixes checks, should be merged immediately.

## Checklist
- [x] The changes pass pre-commit checks (`make lint`)
- [x] The changes follow coding style

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ repos:
     rev: 1.13.0
     hooks:
     -   id: blacken-docs
-        additional_dependencies: [black==21.11b1]
```
Comment (cmyui) [inline .pre-commit-config.yaml:36]: can't we just set it to 22.12.0 (like is used above)? this *is* an additional dependency that's required for this hook, i don't think we should remove it outright

Comment (NiceAesth) [reply .pre-commit-config.yaml:36]: i see VERY little point in adding the dependency there when black itself is already in the same file as a dependency. this is just a liability that we have to maintain on every update at this point

Comment (cmyui) [reply .pre-commit-config.yaml:36]: fair

**Review by cmyui — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (NiceAesth) [reply .pre-commit-config.yaml:36]: i would be fine with adding it back if pre-commit ci would handle it but it doesn't sadly

**Pull request merged**